src/transport/SessionManager.cpp

```diff
@@ -601,13 +601,16 @@ void SessionManager::SecureGroupMessageDispatch(const PacketHeader & packetHeade
     PayloadHeader payloadHeader;
     SessionMessageDelegate::DuplicateMessage isDuplicate = SessionMessageDelegate::DuplicateMessage::No;
     Credentials::GroupDataProvider * groups              = Credentials::GetGroupDataProvider();
+    GroupId groupId;
     VerifyOrReturn(nullptr != groups);
 
     if (!packetHeader.GetDestinationGroupId().HasValue())
     {
         return; // malformed packet
     }
 
+    groupId = packetHeader.GetDestinationGroupId().Value();
```

Comment:
Why not declare it here?  This is not C, and we are not using `exit:` stuff in this function.

Reply:
Done